### PR TITLE
Args prompt for sample_enroll

### PR DESCRIPTION
**Pull request by 0xc0dedad**

### CODE REVIEW

File: programs/sample_enroll/sample_enroll.c [hunk 1]
```diff
@@ -75,6 +75,12 @@ struct enroll_cert {
 #include <ziti/zitilib.h>
 
 int main(int argc, char **argv) {
+
+    if (argc != 2) {
```
Comment (ekoby) [inline programs/sample_enroll/sample_enroll.c:79]: this should be `argc >= 3`

usage is `sample_enroll <JWT file> <ID file> [key_file] [cert_file]`

**Review by ekoby — CHANGES_REQUESTED** [review]
see comments

File: programs/sample_enroll/sample_enroll.c (hunk 1 above)
Comment (0xc0dedad) [reply programs/sample_enroll/sample_enroll.c:79]: You probably have meant `if (argc  < 3) -> prompt`?
JWT/ID files are mandatory.
key/cert files are optional.

Comment (0xc0dedad) [reply programs/sample_enroll/sample_enroll.c]: Fixed.

**Review by ekoby — APPROVED** [review]

**Comment by ekoby** [comment]
Thank you for your contribution

**Pull request merged**